src/coreComponents/physicsSolvers/contact/SolidMechanicsAugmentedLagrangianContact.cpp

```diff
@@ -308,31 +345,113 @@ void SolidMechanicsAugmentedLagrangianContact::assembleSystem( real64 const time
 
     string const & fractureRegionName = this->getUniqueFractureRegionName();
 
-    forFiniteElementOnFractureSubRegions( meshName, [&] ( string const & finiteElementName,
-                                                          arrayView1d< localIndex const > const & faceElementList )
+    forFiniteElementOnStickFractureSubRegions( meshName, [&] ( string const &,
+                                                               finiteElement::FiniteElementBase const & subRegionFE,
+                                                               arrayView1d< localIndex const > const & faceElementList,
+                                                               bool const )
     {
 
-      finiteElement::FiniteElementBase & subRegionFE = *(m_faceTypeToFiniteElements[finiteElementName]);
+      if( m_simultaneous )
```

Comment:
maybe we can do something similar to what I did here: https://github.com/GEOS-DEV/GEOS/pull/3209/files#diff-1f195d2cb02c8766e6929f0ee5e87bfcf4fbec22fe7e1c392a3e927a1db478f2 
to get the correct factory?